README.rst

```diff
@@ -0,0 +1,65 @@
+======================================
+MapR topology for clusterdock
+======================================
+
+This repository houses the **MapR** topology for `clusterdock`_.
+
+.. _clusterdock: https://github.com/clusterdock/clusterdock
+
+Usage
+=====
+
+Assuming you've already installed **clusterdock** (if not, go `read the docs`_),
+you use this topology by cloning it to a local folder and then running commands
+with the ``clusterdock`` script:
+
+.. _read the docs: http://clusterdock.readthedocs.io/en/latest/
+
+.. code-block:: console
+
+    $ git clone https://github.com/clusterdock/topology_mapr.git
+    $ clusterdock start topology_mapr --namespace streamsets --node-disks='{node-1:[/dev/xvdb],node-2:[/dev/xvdc]}' --predictable --mapr-version=5.2.0
```

Comment:
I'd replace the equal signs between `--arg` and the argument itself with spaces. Using equals can get hairy in a few edge cases that we actually encounter (e.g. when passing node names to `--secondary-nodes`).

Reply:
Done.